packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -470,6 +605,14 @@ class DaemonDomain extends Domain {
   }
 }
 
+/// The reason a [PlatformType] is not currently supported.
+///
+/// The [name] of this value will be sent as a response to daemon client.
```

Comment:
nit: I would call out specifically that (despite being a private enum) clients may be using these names and that they shouldn't be changed.

(and maybe a short description on each member explaining what it means, since clients may want to make decisions based on it)